docs/submit/osg-flock.md

```diff
@@ -1,97 +1,170 @@
-using [these instructions](/security/host-certs/)
 
-**Optional** Pool Password: HTCondor Configuration
-------------------------------------------------
+Configuring Authentication
+--------------------------
+If you have obtained a host certificate according to the [instructions](/security/host-certs.md),
+you do not need to do any additional configuration and may skip the rest of this section.
 
-This section is optional and should be only considered if the local admin cannot obtain
-a host certificate for their submit host. To enable pool password authentication place a file here:
-`/etc/condor/config.d/90-flock-pool-password.conf` with the following contents.
+If you cannot obtain a host certificate, then configure pool password authentication via the following instructions:
 
+1. Send email to <help@opensciencegrid.org> requesting the pool password file for flocking.
 
-```file
-   #-- Flock to the OSG using pool password
-   SEC_PASSWORD_FILE = /etc/condor/pool_password
-   SEC_DEFAULT_AUTHENTICATION_METHODS = FS,PASSWORD
-```    
+1. Place the pool password file in `/etc/condor/pool_password`.
 
+1. Create a configration file named `/etc/condor/config.d/90-flock-pool-password.conf` with the following contents:
 
+        :::ini
+        #-- Flock to OSG using pool password
+        SEC_PASSWORD_FILE = /etc/condor/pool_password
+        SEC_DEFAULT_AUTHENTICATION_METHODS = FS,PASSWORD
 
-Project Names
--------------
 
-OSG will only run jobs tagged with a valid *ProjectName* - this is the main attribute
-used for accounting. Please open at ticket to register a new project.
-Jobs should specify which project to be accounted against by adding
-the *+ProjectName* attribute. Note that the value is a string and hence
-the double quotes are required. For example:
+Managing Services
+-----------------
+You will need the following services enabled and running:
+* gratia-probes-cron
+* condor
+
+The following table gives the commands needed to start, stop, enable, and disable a service:
+
+To...                                       | On EL6, run the command...              | On EL7, run the command...         |
+:------------------------------------------ | :-------------------------------------- | :--------------------------------- |
+| Start a service                           | `service <SERVICE-NAME> start`          | `systemctl start <SERVICE-NAME>`   |
+| Stop a service                            | `service <SERVICE-NAME> stop`           | `systemctl stop <SERVICE-NAME>`    |
+| Enable a service to start on boot         | `chkconfig <SERVICE-NAME> on`           | `systemctl enable <SERVICE-NAME>`  |
+| Disable a service from starting on boot   | `chkconfig <SERVICE-NAME> off`          | `systemctl disable <SERVICE-NAME>` |
+
+
+Usage
+-----
+### Running jobs in OSG
+Users should be aware that OSG jobs are distributed across multiple institutions across a large geographical area.
```

Comment:
[Add as first sentence:] If your users are accustomed to running jobs locally, they may encounter some significant differences when running jobs in OSG.